.changeset/curly-suns-explain.md

```diff
@@ -0,0 +1,5 @@
+---
+'@axelar-network/axelar-cgp-sui': major
```

Comment:
Change the top level version.json so gh actions use the correct one (hard to see on the phone if that was changed)